src/emulator/functionsEmulator.ts

```diff
@@ -566,16 +566,10 @@ You can probably fix this by running "npm install ${
   addRealtimeDatabaseTrigger(
     projectId: string,
     definition: EmulatedTriggerDefinition
-  ): Promise<any> {
```

Comment:
Keep an eye out for any of these `<any>` which should actually be `<void>`. This fix is right, I just think there are other excessive `any`